index.html

```diff
@@ -5041,16 +5041,19 @@ <h5>Note regarding the ARIA 1.1 <rref>none</rref> role.</h5>
 				<section id="conflict_resolution_presentation_none">
 					<h5>Presentational Roles Conflict Resolution</h5>
 					<p>There are a number of ways presentational role conflicts are resolved.</p>
-					<p>Host languages elements having implicit presentational roles for which no roles may be applied MUST never be exposed in the accessibility tree, with this exception: user agents MUST always expose global WAI-ARIA states and properties to accessibility APIs. In this case, the user agent ignores the <code>presentation</code> role and exposes the element according to its implicit native semantics. However, user agents MUST ignore any non-global, role-specific WAI-ARIA states and properties, unless it is on an inherited presentational role where an explicit role is applied.</p>
-					<p>For example, <pref>aria-haspopup</pref> is a global attribute and would always be applied; <pref>aria-level</pref> is not a global attribute and would therefore only apply if the element was not in a presentational state.</p>
+					<p>Host language elements having explicit or inherited presentational role MUST NOT be exposed in the accessibility tree, with these exceptions:</p>
+					<ul>
+						<li>If an element is focusable, or otherwise interactive, user agents MUST ignore the <code>presentation</code> role and expose the element as described in the accessibility API mappings for presentational objects that remain in the accessibility tree, in order to ensure that the element is both <a>understandable</a> and <a>operable</a>.</li>
+						<li>If a <a href="#mustContain">required owned element</a> has an explicit non-presentational role, user agents MUST ignore an inherited presentational role and expose the element with its explicit role. If the action of exposing the explicit role causes the accessibility tree to be malformed, the expected results are undefined and the user agent MAY resort to an internal recovery mechanism to repair the accessibility tree.</li>
+						<li>If an element has global WAI-ARIA states or properties, user agents MUST ignore the <code>presentation</code> role and expose the element as described in the accessibility API mappings for presentational objects that remain in the accessibility tree. However, if an element has only non-global, role-specific WAI-ARIA states or properties, the element MUST NOT be exposed unless the presentational role is inherited and an explicit non-presentational role is applied.</li>
```

Comment:
Remove mention of API mappings.

Reply:
So, remove " as described in the accessibility API mappings for presentational objects that remain in the accessibility tree" so that the sentence reads:
> If an element has global WAI-ARIA states or properties, user agents MUST ignore the <code>presentation</code> role and expose the element.

Reply:
Yes.